### PR TITLE
Rework of the network page

### PR DESCRIPTION
**Pull request by annando**
The network page is reworked. This is a preparation for - at one time - converting the network to a module. This is just an intermediate step. The next step will be creating an item view. Then the main query will be completely rebuild. After this is does make sense to convert it into a module - but not before.

### CODE REVIEW

File: mod/network.php [hunk 1]
```diff
@@ -518,25 +499,20 @@ function networkThreadedView(App $a, $update, $parent)
...
+	$sql_extra = ($star ? " AND `thread`.`starred` " : '') .
+		($conv ? " AND $sql_table.`mention`" : '') .
+		($nets ? sprintf(" AND $sql_table.`network` = '%s' ", DBA::escape($nets)) : '');
 
-	if ($update) {
-		$sql_table = '`item`';
-		$sql_parent = '`parent`';
-		$sql_post_table = " INNER JOIN `thread` ON `thread`.`iid` = `item`.`parent`";
+	if ($datequery) {
+		$sql_extra .= Strings::protectSprintf(sprintf(" AND $sql_table.received <= '%s' ",
```
Comment (MrPetovan) [inline mod/network.php:509]: Please use `DBA::mergeConditions` instead of these `sprintf()` calls.

Comment (annando) [reply mod/network.php:509]: No, not at this stage of the process. This here had just been some cleaning up. The switch to using `DBA` functions will be done when the item view(s) had been created. This PR here is just for testing if my simplifications are okay.

Comment (MrPetovan) [reply mod/network.php:509]: As long as you have it in the back of your mind.

Comment (annando) [reply mod/network.php:509]: My whole rework is based on replacing those two `q(` queries. For this the structure needs to be simplified. This step is done. Now I hope that anyone does notice any side effects. Then I can create that view and perform this whole parameter magic.

Question: Would you then like to continue by converting this into a module or should I finish it completely?

Comment (MrPetovan) [reply mod/network.php:509]: I'm busy with CSS that I know you will not pick up if I put is aside, so I'd prefer if you did it yourself. I'll be providing feedback from the sidelines.

**Pull request merged**